install.sh

```diff
@@ -229,16 +230,16 @@ function download_check () {
     cd "${CREW_BREW_DIR}"
 
     #download
-    echo "Downloading ${1}..."
+    echo -e "${BLUE}Downloading ${1}...${RESET}"
     curl --progress-bar -C - -L --ssl "${2}" -o "${3}"
 
     #verify
-    echo "Verifying ${1}..."
-    echo "${4}" "${3}" | sha256sum -c -
+    echo -e "${BLUE}Verifying ${1}..."
+    echo -e ${GREEN}`echo "${4}" "${3}" | sha256sum -c -`${RESET}
     case "${?}" in
     0) ;;
     *)
-      echo "${RED}Verification failed, something may be wrong with the download."
+      echo -e "${RED}Verification failed, something may be wrong with the download."
```

Comment:
Missing `${RESET}` at the end of the string?

Reply:
Done

Reply:
Did I miss the change?